code/game/rollcredits.dm

```diff
@@ -0,0 +1,184 @@
+	var/list/possible_titles = list()
+	var/chunksize = 0
+
+	/* Establish a big-ass list of potential titles for the "episode". */
+	possible_titles += "THE [pick("DOWNFALL OF", "RISE OF", "TROUBLE WITH", "FINAL STAND OF", "DARK SIDE OF")] [pick("SPACEMEN", "HUMANITY", "DIGNITY", "SANITY", "THE CHIMPANZEES", "THE VENDOMAT PRICES","[uppertext(station_name())]")]"
+	possible_titles += "THE CREW GETS [pick("RACIST", "PICKLED", "AN INCURABLE DISEASE", "PIZZA", "A VALUABLE HISTORY LESSON", "A BREAK", "HIGH", "TO LIVE", "TO RELIVE THEIR CHILDHOOD", "EMBROILED IN CIVIL WAR", "SERIOUS ABOUT [pick("DRUG ABUSE", "CRIME", "PRODUCTIVITY", "ANCIENT AMERICAN CARTOONS", "SPACEBALL")]")]"
+	possible_titles += "THE CREW LEARNS ABOUT [pick("LOVE", "DRUGS", "THE DANGERS OF MONEY LAUNDERING", "SPACE 'NAM", "INVESTMENT FRAUD", "KELOTANE ABUSE", "RADIATION PROTECTION", "SACRED GEOMETRY", "STRING THEORY", "ABSTRACT MATHEMATICS", "[pick("CATBEAST", "DIONAN", "PLASMAMAN", "VOX", "GREY")] MATING RITUALS", "ANCIENT CHINESE MEDICINE")]"
+	if(SNOW_THEME)
+		possible_titles += "A VERY [pick("NANOTRASEN", "EXPEDITIONARY", "DIONA", "PLASMA", "MARTIAN")] CHRISTMAS"
+	possible_titles += "[pick("GUNS, GUNS EVERYWHERE", "MUCH ADO ABOUT NOTHING", "WHAT HAPPENS WHEN YOU MIX MOMMIS AND COMMERCIAL-GRADE PACKING FOAM", "ATTACK! ATTACK! ATTACK!", "SEX BOMB", "THE BALLAD OF [uppertext(station_name())]")]"
+	possible_titles += "[pick("SPACE", "SEXY", "DRAGON", "WARLOCK", "LAUNDRY", "GUN", "ADVERTISING", "DOG", "CARBON MONOXIDE", "NINJA", "WIZARD", "SOCRATIC", "JUVENILE DELIQUENCY", "POLITICALLY MOTIVATED", "RADTACULAR SICKNASTY")] [pick("QUEST", "FORCE", "ADVENTURE")]"
+	possible_titles += "[pick("THE DAY [uppertext(station_name())] STOOD STILL", "HUNT FOR THE GREEN WEENIE", "ALIEN VS VENDOMAT", "SPACE TRACK")]"
+	titles += "<center><h1>SEASON [rand(1,22)] EPISODE [rand(1,17)]<br>[pick(possible_titles)]<h1></h1></h1></center>"
+	titles += "<center><h1>CAST:</h1><h2>(in order of appearance)"
+	for(var/mob/living/carbon/human/H in living_mob_list|dead_mob_list)
+		if(!star || H.talkcount > star.talkcount)
+			star = H
+		if(H.timeofdeath && H.timeofdeath < 5 MINUTES) //don't mention these losers (prespawned corpses mostly)
+			continue
+
+		chunk += gender_credits(H)
+		chunksize++
+		if(chunksize == 5)
+			cast += chunk
+			chunk = "<center>"
+			chunksize = 0
+	titles += cast
+	if(chunksize)
+		titles += chunk
+
+	var/list/corpses = list()
+	for(var/mob/living/carbon/human/H in dead_mob_list)
+		if(H.timeofdeath < 5 MINUTES) //no prespawned corpses
+			continue
+		else if(H.real_name)
+			corpses += H.real_name
+	if(corpses.len)
+		titles += "<center>BASED ON REAL EVENTS<br>In memory of [english_list(corpses)].</center>"
+
+	var/list/staff = list("<h1>PRODUCTION STAFF</h1>")
+	var/list/staffjobs = list("Coffee Fetcher", "Cameraman", "Angry Yeller", "Chair Operator", "Choreographer", "Historical Consultant", "Costume Designer", "Chief Editor", "Executive Assistant")
+	for(var/client/C)
```

Comment:
Is there no list of admins that can be used instead?

Reply:
Is there?